cmd/kubeadm/app/phases/upgrade/configuration.go

```diff
@@ -52,6 +53,29 @@ func FetchConfiguration(client clientset.Interface, w io.Writer, cfgPath string)
 	return versionedcfg, nil
 }
 
+// FetchConfigurationFromFile fetch configuration from a file
+func FetchConfigurationFromFile(cfgPath string) (*kubeadmapiext.MasterConfiguration, error) {
+	fmt.Println("[upgrade/config] FetchConfiguration From File.")
+
+	if cfgPath == "" {
```

Comment:
Is this logic necessary or is this case redundant as it's handled inside of ReadFile()?